project/Dependencies.scala

```diff
@@ -1,28 +1,34 @@
 import sbt._
 
 object Version {
-  val spark        = "1.4.0"
+  val spark             = "1.4.0"
 }
 
 object Library {
-  val sparkCore      = "org.apache.spark"  % "spark-core_2.11" % Version.spark
-  val scalaTest      = "org.scalatest" % "scalatest_2.11" % "2.2.4"
-  val adam = "org.bdgenomics.adam" % "adam-core_2.11" % "0.17.0"
-  val cli = "org.bdgenomics.utils" % "utils-cli_2.11" % "0.2.2"
-  val utils = "org.bdgenomics.utils" % "utils-misc_2.11" % "0.2.2"
-  val algebird = "com.twitter" % "algebird-core_2.11" % "0.10.2"
+  val sparkCore         = "org.apache.spark" %% "spark-core" % Version.spark
+  val sparkMLlib        = "org.apache.spark" %% "spark-mllib" % Version.spark
```

Comment:
What is MLlib being used for in the code? I've seen it reference, but I'm not sure it's necessary at this point?